hail/python/hail/docs/makeDocs.sh

```diff
@@ -17,8 +17,19 @@ xsltproc --html -o build/www/jobs.html www/jobs.xslt build/tmp/jobs.html
 pandoc -s www/about.md -f markdown -t html --mathjax --highlight-style=pygments --columns 10000 -o build/tmp/about.html
 xsltproc --html -o build/www/about.html www/about.xslt build/tmp/about.html
 
+cleanup() {
+    set +e
```

Comment:
Some folks spam ctrl-c until a script dies, if you get rapid fire INTs/TERMs your cleanup won't execute, so I start an exit trap handler with:

```
trap "" INT TERM
```
which means "do nothing" when INT or TERM is received

Reply:
these comments were helpful, thanks!